### DISCUSSION TITLE
Unable to verify student benefits

### DISCUSSION THREAD

**Discussion by berthdev** [community/community#53151] [GitHub Education]
Hello,

I would like to share my experience and maybe you can advise me. I am writing here after my fourth rejected application.

1. I provided my International Student Identity Card, and it was rejected. It is a valid ISIC student card with the name of the school, validity date, my name and ISIC card number, and other information that you could easily verify using the barcode or otherwise.

2. I provided all the necessary information that was required of me after the first rejection. The second request was also rejected with this link: 

https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved

3. I provided a Confirmation of School Visit (in my language) with a stamp and signature of the school employee. It was also rejected. The rejection came within 10 minutes.

4. In our country, we have this website where all the students are listed, and it can be easily verified: https://portal.edupage.org/ Using this website, My parents created a Confirmation of School Visit in English, with a unique QR code that can be verified on the official website. This is an official application that is used by students throughout the country, and I think it would be good for verifying future applicants from my country. However, I still received a rejection response within 10 minutes.

I don't know what other information I could provide or why I were rejected. Because I am getting only this link mentioned above again and again.

Thank you very much.

**Comment by samus-aran** [comment]
Hi @berthdev 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Hope this helps! Let us know how you get along 👍

**Reply by er844** [reply]
Have you read what he wrote?
